helpers/middleman/url.rb

```diff
@@ -154,4 +156,31 @@ def strip_pagination_from_path(path)
   def image_url(source)
     return absolute_url(image_path(source))
   end
+
+  def is_remote_url?(url)
```

Comment:
@irwalker My Ruby reviewer, I think this was a bit of overkill to get what I wanted…but it felt like a good thing to add.

1. We should open in a new tab when we're going to a non-local path/url.
2. We should use `rel="noopener noreferrer"` or whatever it is for things outside of our ecosystem.

See commit: d40ff12b73d4d62b6da9090043b197a8b7e03536

Reply:
Makes sense to me